lib/usdUfe/ufe/Utils.h

```diff
@@ -42,6 +43,7 @@ typedef Ufe::Path (*StagePathAccessorFn)(PXR_NS::UsdStageWeakPtr);
 typedef PXR_NS::UsdPrim (*UfePathToPrimFn)(const Ufe::Path&);
 typedef PXR_NS::UsdTimeCode (*TimeAccessorFn)(const Ufe::Path&);
 typedef bool (*IsAttributeLockedFn)(const PXR_NS::UsdAttribute& attr, std::string* errMsg);
+typedef void (*SaveStageLoadRulesFn)(const PXR_NS::UsdStageRefPtr&);
```

Comment:
Any reason not to use std::function<>?

Reply:
I recall that Pierre and I talked about it and decided to just use regular function pointers. We don't need the extra indirection of std::function and exception.